### PR TITLE
feat(webpack): added source maps support for deps in development mode

### PR DESCRIPTION
**Pull request by giulianok**
This add source maps rule to get the nice pretty code from libraries that have source maps

### CODE REVIEW

File: packages/one-app-bundler/__tests__/webpack/app/__snapshots__/webpack.client.spec.js.snap [hunk 1]
```diff
@@ -22,39 +22,74 @@ Object {
...
-    "/src",
-  ],
-  "test": /\\\\\\.jsx\\?\\$/,
-  "use": Array [
-    Object {
-      "loader": "babel-loader",
-      "options": Object {
-        "envName": undefined,
-        "extends": "/.babelrc",
+  "enforce": "pre",
```
Comment (JAdshead) [inline packages/one-app-bundler/__tests__/webpack/app/__snapshots__/webpack.client.spec.js.snap:25]: I don't think these should have changed.

File: packages/one-app-bundler/webpack/webpack.common.js [hunk 2]
```diff
@@ -73,6 +73,11 @@ module.exports = {
           },
         }],
       },
+      !nodeEnvIsProduction && {
```
Comment (JAdshead) [inline packages/one-app-bundler/webpack/webpack.common.js:76]: can you add a test ensuring that this does not make it into production and a test that shows it is included

**Comment by JamesSingleton** [comment]
@giulianok, you need to resolve the conflict.

File: packages/one-app-bundler/__tests__/webpack/app/__snapshots__/webpack.client.spec.js.snap (hunk 1 above)
Comment (giulianok) [reply packages/one-app-bundler/__tests__/webpack/app/__snapshots__/webpack.client.spec.js.snap:25]: this is autogenerated :man-shrugging:

File: packages/one-app-bundler/webpack/webpack.common.js (hunk 2 above)
Comment (giulianok) [reply packages/one-app-bundler/webpack/webpack.common.js:76]: added

**Comment by JAdshead** [comment]
closing due to lack of activity, feel free to open if you feel it is still relevant

**Pull request closed without merging**